extensions/amp-story/1.0/page-advancement.js

```diff
@@ -42,6 +42,13 @@ const NEXT_SCREEN_AREA_RATIO = 0.75;
 /** @private @const {number} */
 const PREVIOUS_SCREEN_AREA_RATIO = 0.25;
 
+/**
+ * Percentage of the screen considered edge.
+ * @const {number}
+ * @private
+ */
+const SCREEN_EDGE_PERCENT = 0.15;
```

Comment:
How did you come up with this number? What do the other Stories platform use here?

I was trying to see how it looks on various devices, it's fine for most portrait devices, but looks really big on landscape devices:

iPhone 5
![image](https://user-images.githubusercontent.com/1492044/57105545-0a7afe80-6cf9-11e9-9b77-36664eec3390.png)

iPhone Plus
![image](https://user-images.githubusercontent.com/1492044/57105573-18308400-6cf9-11e9-8249-50763000b761.png)

iPhone Plus landscape
![image](https://user-images.githubusercontent.com/1492044/57105583-1ff02880-6cf9-11e9-8c06-ab5a5b3c3227.png)

Laptop 1366 * 738
![image](https://user-images.githubusercontent.com/1492044/57105600-30080800-6cf9-11e9-8873-136fa562d738.png)

Reply:
I calculated the percentage that we currently use for the navigation arrows, when displayed on a tooltip. Do you prefer a static number of pixels?